dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/dublincore/DublinCoreExportUtil.java

```diff
@@ -48,14 +46,18 @@
     public static String DEFAULT_DC_FLAVOR = DC_FLAVOR_DCTERMS;
 
 
-    public static void datasetJson2dublincore(JsonObject datasetDtoAsJson, OutputStream outputStream, String dcFlavor) throws XMLStreamException {
-        logger.fine(JsonUtil.prettyPrint(datasetDtoAsJson.toString()));
+    public static void datasetJson2dublincore(String datasetDtoAsJson, OutputStream outputStream, String dcFlavor) throws XMLStreamException {
```

Comment:
Didn't we get rid of versions that uses `String` or `JsonObject` as `DatasetDTO`?